app/uk/gov/hmrc/incometaxpenaltiesfrontend/views/components/PenaltiesOverview.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxpenaltiesfrontend.viewModels.PenaltiesOverviewViewModel
+@import uk.gov.hmrc.incometaxpenaltiesfrontend.views.html.components
+@import uk.gov.hmrc.incometaxpenaltiesfrontend.config.AppConfig
+
+@this(
+        list: components.List,
+        h2: components.H2,
+        p: components.P,
+        link: components.Link,
+        appConfig: AppConfig
+)
+
+@(penaltiesOverviewViewModel: PenaltiesOverviewViewModel, isAgent: Boolean)(implicit messages: Messages)
+
+@if(penaltiesOverviewViewModel.content.nonEmpty) {
+    @defining(if(isAgent) "agent" else "individual") { individualOrAgent =>
+        <div class="govuk-form-group" id="penaltiesOverview">
```

Comment:
govuk-form-group class is not needed here. It increases the margin bottom to 30px which is more than whats in the prototype 
![Screenshot 2025-02-03 at 13 04 18](https://github.com/user-attachments/assets/ff822784-1997-4f70-84e3-565e8fa5bc76)